$OpenBSD: patch-configure,v 1.3 2012/03/19 08:59:00 ajacoutot Exp $

Fix DATADIRNAME: see LOCALBASE/share/aclocal/glib-gettext.m4.

/var/lib -> /var/db

--- configure.orig	Thu Mar 15 12:13:39 2012
+++ configure	Sun Mar 18 20:28:58 2012
@@ -13555,6 +13555,9 @@ else
 fi
 
     ;;
+    *-*-openbsd*)
+    DATADIRNAME=share
+    ;;
     *)
     DATADIRNAME=lib
     ;;
@@ -16804,7 +16807,7 @@ fi
 
 
 # per-machine directory
-CD_SYSTEM_PROFILES_DIR="\$(localstatedir)/lib/colord/icc"
+CD_SYSTEM_PROFILES_DIR="\$(localstatedir)/db/colord/icc"
 
 
 ac_config_files="$ac_config_files Makefile etc/Makefile contrib/Makefile contrib/colord-sane/Makefile contrib/colord-sane/org.freedesktop.colord-sane.conf examples/Makefile man/Makefile doc/Makefile doc/api/Makefile doc/api/dbus/Makefile doc/api/version.xml data/Makefile data/org.freedesktop.ColorManager.conf data/tests/Makefile policy/Makefile rules/Makefile rules/69-cd-sensors.rules po/Makefile.in src/Makefile src/sensors/Makefile client/Makefile libcolord/cd-version.h libcolord/colord.pc libcolord/Makefile"
